src/test/java/redis/clients/jedis/tests/commands/AllKindOfValuesCommandsTest.java

```diff
@@ -355,14 +360,33 @@ public void ttl() {
 
   }
 
+  private Callable<Boolean> jedisTargetIsIdle(final String target) {
+    return new Callable<Boolean>() {
+      @Override
+      public Boolean call() throws Exception {
+        return jedis.objectIdletime(target) > 0;
+      }
+    };
+  }
+
+  private Callable<Boolean> jedisTargetIsIdle(final byte[] target) {
+    return new Callable<Boolean>() {
+      @Override
+      public Boolean call() throws Exception {
+        return jedis.objectIdletime(target) > 0;
+      }
+    };
+  }
   @Test
   public void touch() throws Exception {
-    long reply = jedis.touch("foo1", "foo2", "foo3");
+    final String foo1 = "foo1";
+    long reply = jedis.touch(foo1, "foo2", "foo3");
     assertEquals(0, reply);
 
-    jedis.set("foo1", "bar1");
+    jedis.set(foo1, "bar1");
+
+    await().atMost(2, TimeUnit.SECONDS).until(jedisTargetIsIdle(foo1));
```

Comment:
The idea here is to wait for at least 1 second. 2000ms is used to be safe. Is at most 2 seconds safe here?